source/MaterialXRenderGlsl/CMakeLists.txt

```diff
@@ -22,11 +18,18 @@ if(APPLE)
     set(materialx_source ${materialx_source} ${materialx_source_oc})
     add_compile_options(-Wno-deprecated-declarations -DGL_SILENCE_DEPRECATION)
 elseif(UNIX)
+    find_package(PkgConfig REQUIRED)
     find_package(X11 REQUIRED)
-    find_package(OpenGL REQUIRED)
+    if(NOT X11_FOUND)
+        message("Error in building MaterialXRender: X11 was not found")
+    endif(NOT X11_FOUND)
     if(NOT X11_Xt_FOUND)
         message("Error in building MaterialXRender: Xt was not found")
-    endif()
+    endif(NOT X11_Xt_FOUND)
+    find_package(OpenGL REQUIRED)
+    if(NOT OPENGL_FOUND)
+        message("Error in building MaterialXRender: OpenGL was not found")
+    endif(NOT OPENGL_FOUND)
```

Comment:
This change may be an unintended reversion, since these calls were simplified in the master branch a while back.  In theory, we shouldn't need to check for `OPENGL_FOUND` or `X11_FOUND` after calling find_package on `OpenGL` and `X11` with the `REQUIRED` flag.

Reply:
Yes. This looks like a synchronization issue on our side. I'll revert this part.